divisible.rb

```diff
@@ -0,0 +1,15 @@
+def divisible
+  divisible = []
+  for item in 1..100 do
+    if item % 2 == 0
+      divisible.push(item)
+    elsif item % 3 == 0
+      divisible.push(item)
+    elsif item % 5 == 0
+      divisible.push(item)
```

Comment:
good job. This looks good. I like how you pushed each number to the divisible array. There is another method that can be used which is just **<<**, called the shovel operator.
what if you added a puts statement after each "divisible.push(item)" line and said, "x number is divisible by 2" ??
I think that would make it clear which number is divisible by 2 or 3 or 5.
as an improvement, what if a number is divisible by more than one number from the 3 we have?